CONTRIBUTING.md

```diff
@@ -9,7 +9,7 @@ prepared by `tox`, including virtual environment setup, dependency management, i
 
 1. Install `tox`:
 
-    $ pip install -U tox  # If you are inside a virtual environment, conda environment
+    $ pip install -U -r requirements/tox.txt  # If you are inside a virtual environment, conda environment
     $ pip3 install --user -U tox  # If you are outside any virtual environment or conda environment
```

Comment:
Do we want to install from `requirements/tox.txt` when outside of a virtual env?

Reply:
Outside an virtual environment is sort of a user's global installation that we better not impose such a restriction though

Reply:
I updated this line to install our specified version, but appended additional note to the comment (no need to run this line if it's a user's global installation)

Reply:
gotcha sounds good